readme.md

```diff
@@ -70,6 +70,25 @@ pify(request, {multiArgs: true})('https://sindresorhus.com').then(result => {
 });
 ```
 
+##### allArgs
+
+Type: `boolean`
+Default: `false`
+
+By default, the promisified function will only return the second argument from the callback. Turning this on will make it return an array of all arguments from the callback, *including* the error argument. This can be useful if you want to avoid a try-catch block when using async-await.
```

Comment:
Also it allows for more 'sequential looking' code if you have multiple `await`s. (Wrapping them in try-catch one by one seems a bit ugly but maybe that's just a matter of personal taste).

Happy to remove that last bit from the documentation if confusing but would really like to get the `allArgs` option added.